src/lua-runtime-fastpm.lua

```diff
@@ -108,10 +108,12 @@ end
 -- Initial condition --
 schema.declare{name='read_lineark',        type='string', help='lineark for cdm'}
 schema.declare{name='read_powerspectrum', type='file', help='file to read the linear power spectrum for cdm.'}
+schema.declare{name='read_linear_growth_rate', type ='file', help='file to read te linear growth rate (f_1) of cdm. If left empty it will use internal scale independent f_1.'}
```

Comment:
Is this file a function of time or scale? (e.g. scale-dependent linear growth rate). What redshift -- does lienar_density_redshift cover it? ( I think it shall)

Reply:
A function of scale. Same form as read_powerspectrum, and also evaluated at linear_density_redshift as you say.